src/pipe_wrap.h

```diff
@@ -38,6 +38,11 @@ class PipeWrap : public ConnectionWrap<PipeWrap, uv_pipe_t> {
     IPC
   };
 
+  enum PipeChmod {
+    READABLE_ALL = UV_READABLE,
+    WRITABLE_ALL = UV_WRITABLE
+  };
```

Comment:
Is there any reason not to expose the libuv names directly? Makes things a bit more `grep`able :)